src/ui/handlers/command-ui-handler.ts

```diff
@@ -261,9 +265,9 @@ export class CommandUiHandler extends UiHandler {
   protected override clear(): void {
     this.commandsContainer.setVisible(false);
     const messageHandler = this.getUi().getMessageHandler();
-    if (messageHandler.ready) {
-      messageHandler.commandWindow.setVisible(false);
-      messageHandler.clearText();
+    if (messageHandler?.ready) {
+      messageHandler?.commandWindow.setVisible(false);
+      messageHandler?.clearText();
```

Comment:
are those `?` necessary? I think the `if` before should cover telling the compiler it's set

```suggestion
      messageHandler.commandWindow.setVisible(false);
      messageHandler.clearText();
```